### ISSUE TITLE
Font size for quote when nitting.

### ISSUE THREAD

**Issue by remcopeereboom** [exercism/exercism.io#2553]
I'm glad some one changed the quotes to be less screamy, but when writing a nit (i.e. in the editor preview), the quote font size is still large and ink black instead of the new smaller and light grey posted quotes.

I looked up the values for the [comment blockquote](https://github.com/exercism/exercism.io/blob/d94c82cb5e7571341d9977a5dedd97916406fedc/lib/app/public/sass/base/components/comment.scss):
````scss
blockquote {
    font-size: 1em;
    color: gray;
    font-style: italic;
  }
````

You can add a nesting under `.md-markdown-preview` in the [submission layout file](https://github.com/exercism/exercism.io/blob/402154b157abd6619f69aaa37e86b83c5c2eca9e/lib/app/public/sass/layouts/submission.scss):

````scss
  .md-markdown-preview {
    a {
      cursor: pointer;
    }
    .preview {
      margin: 10px 0px;
    }
    blockquote {
      font-size: 1em;
      color: gray;
      font-style: italic;
    }
  }
````

But you probably want to add some import so the discrepancy doesn't happen again. I'm not familiar enough with the code base to know where you keep those base pieces, hence this issue instead of  a pull request.

**Comment by kytrinyx** [comment]
Thanks! I've been kind of hacking away at the front-end, and I don't actually know front-end development, which is why we're in this muddle. Maybe @mixolidia and @sguermond know how the imports work and could take a look at it.

**Comment by kytrinyx** [comment]
@GustavoCaso would you like to take a look at this one?

**Comment by GustavoCaso** [comment]
Sure @kytrinyx no problem ! Thanks

**Comment by kytrinyx** [comment]
Fixed in #2622
